nikola/conf.py.in

```diff
@@ -89,6 +89,10 @@ NAVIGATION_LINKS = ${NAVIGATION_LINKS}
 # Name of the theme to use.
 THEME = ${THEME}
 
+# Primary color of your theme. This will be used to customize your theme and
+# auto-generate related colors in POSTS_CATEGORY_COLORS. Must be a HEX value.
+THEME_COLOR = '#5670d4'
```

Comment:
Generic light corporate blue were designed to be non-offensive and enduce trust. Sounds like a perfect candidate for a nonintrusive default if I ever heard one. Goes well with the default bootstrap3 theme. Feel free to suggest another.